static/external/manyplot.js

```diff
@@ -831,7 +831,7 @@ function createPlotter( canvas, multiFrame, options ) {
         }
     };
 
-    plotter.setIntervalFirstBound = function(xMouse){
+    plotter.setIntervalFirstBound = function (xMouse) {
```

Comment:
I want to take a pass during an upcoming tech debt Friday and fix all of this sort of inconsistent spacing.